server/src/main/java/io/crate/execution/ddl/tables/AlterTableClient.java

```diff
@@ -103,7 +100,7 @@ public AlterTableClient(ClusterService clusterService,
         this.clusterService = clusterService;
         this.transportRenameTableAction = transportRenameTableAction;
         this.transportResizeAction = transportResizeAction;
-        this.transportDeleteIndexAction = transportDeleteIndexAction;
+        this.transportGCDanglingArtifactsAction = transportGCDanglingArtifactsAction;
```

Comment:
Maybe also worth a follow up to migrate all the transport actions here to use the node client instead.

Reply:
on it, thx!